### PR TITLE
issue #168 use poi 5.2.2 XLConnect java and deps

### LINKED ISSUE #168: Upgrade to latest version of Apache POI
Upgrade to latest version of Apache POI (5.2.2 as of this writing)

### PR DESCRIPTION
**Pull request by spoltier**
closes #168

### CODE REVIEW

**Comment by Chargothrond** [comment]
I am not familiar with this, but saw that last time we updated this, we also had to update xlconnectjars. So not sure if that is required this time. Anyways, from what I can tell compared to https://github.com/miraisolutions/xlconnect/pull/96/files, the changes are consistent and make sense to me (but Martin or Roland should definitely review and approve as well).

File: R/onLoad.R [hunk 1]
```diff
@@ -58,32 +58,35 @@
   sharedPaths <- tryCatch({
```
Comment (martinstuder) [inline R/onLoad.R:58]: The POI release notes mention some other dependencies such as PDFBox graphics2d but I guess we won't need those for the functionality exposed through XLConnect.

File: inst/COPYRIGHTS [hunk 2]
```diff
@@ -2,7 +2,7 @@ The package XLConnect includes other open source software components. The follow
```
Comment (martinstuder) [inline inst/COPYRIGHTS:2]: It really only "includes" (upfront) curvesapi these days and downloads others on-the-fly. We should probably update this statement.

File: inst/COPYRIGHTS [hunk 3]
```diff
@@ -11,12 +11,11 @@ The package XLConnect includes other open source software components. The follow
 
    Related JARs: poi-*.jar, poi-ooxml-*.jar
 
-
 Transitive dependencies include:
```
Comment (martinstuder) [inline inst/COPYRIGHTS:14]: Not sure if we need/should extend the list of transitive dependencies. I guess we could do one line for all Apache projects such as POI, Commons Compress, Commons IO, XMLBeans, etc.

**Review by martinstuder — APPROVED** [review]

File: R/onLoad.R (hunk 1 above)
Comment (spoltier) [reply R/onLoad.R:58]: That seems to be the case, unless we have some functionality that would use it but is not tested ? As far as I know XLConnect doesn't have the ability to generate PDFs.

File: inst/COPYRIGHTS (hunk 3 above)
Comment (spoltier) [reply inst/COPYRIGHTS:14]: I will also remove the "related JARs" as well for the dependencies that are downloaded on-the-fly.

**Comment by spoltier** [comment]
@RolandASc I'll merge this tomorrow if you have not additional comments.

**Review by RolandASc — APPROVED** [review]

**Pull request merged**